packages/glasscoder/PKGBUILD

```
@@ -0,0 +1,28 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+pkgname=glasscoder
+pkgver=1.0.4
+pkgrel=1
+pkgdesc='Minimalist audio encoder for IceCast, Shoutcast and HLS'
+arch=(x86_64 aarch64)
+url='https://github.com/ElvishArtisan/GlassCoder'
+license=(GPL2)
+depends=(curl lame libfdk-aac libogg libsamplerate libsndfile libvorbis openssl opus python qt5-tools taglib twolame)
+makedepends=(alsa-lib docbook-xsl docbook5-xml jack libxslt)
+optdepends=('alsa-lib: for ALSA audio sources'
+            'jack: for JACK audio sources')
```

@thisven Sorry, I just checked and my earlier suggestion was wrong. `jack` and `alsa-lib` are strictly required at runtime, otherwise you get errors like
```
glasscoder: error while loading shared libraries: libjack.so.0: cannot open shared object file: No such file or directory
```
And of course I forgot the `hicolor-icon-theme`. There you go:
```suggestion
depends=(alsa-lib curl hicolor-icon-theme jack lame libfdk-aac libogg libsamplerate libsndfile
  libvorbis openssl opus python qt5-tools taglib twolame)
makedepends=(docbook-xsl docbook5-xml libxslt)
```

---

(The same would be the case if we built with libhpi/AudioScience HPI support 😅)

---

OK, then almost back to the first approach, but including the icon theme package. :sweat_smile: